rust/stackablectl/src/cmds/debug.rs

```diff
@@ -0,0 +1,380 @@
+                        .unwrap_or_default()
+                        || container
+                            .and_then(|c| c.last_state.as_ref()?.terminated.as_ref())
+                            .is_some()
+                },
+            )
+            .await
+            .with_context(|_| AwaitDebugContainerReadinessSnafu {
+                pod: pod_ref(),
+                container: &self.container,
+            })?;
+            let debug_container_status = ready_pod
+                .as_ref()
+                .and_then(debug_container_status_of_pod(&debug_container_name))
+                .with_context(|| FindDebugContainerStatusSnafu {
+                    pod: pod_ref(),
+                    container: &self.container,
+                })?;
+            if let Some(termination) = debug_container_status
+                .last_state
+                .as_ref()
+                .and_then(|state| state.terminated.as_ref())
+            {
+                error!(
+                    error = termination.message,
+                    exit_code = termination.exit_code,
+                    "Debug container failed to start!"
+                );
+            }
+            info!("Attaching to container");
+            let mut attachment = pods
+                .attach(
+                    &self.pod,
+                    &AttachParams::interactive_tty().container(debug_container_name),
+                )
+                .await
+                .with_context(|_| AttachContainerSnafu {
+                    pod: pod_ref(),
+                    container: &self.container,
+                })?;
+            info!("Attached to container, if the shell line looks empty, press ENTER!");
+            let _raw = std::io::stdout()
+                .into_raw_mode()
+                .context(SetRawTtyModeSnafu)?;
+            futures::future::select_all([
+                update_terminal_size(
+                    attachment
+                        .terminal_size()
+                        .context(NoTerminalSizeChannelSnafu)?,
+                )
+                .boxed(),
+                tokio::io::copy(
+                    &mut attachment.stdout().context(NoStdoutChannelSnafu)?,
+                    &mut tokio::io::stdout(),
+                )
+                .map_ok(drop)
+                .context(ForwardStdoutSnafu)
+                .boxed(),
+                tokio::io::copy(
+                    &mut AsyncStdin::new()?,
+                    &mut attachment.stdin().context(NoStdinChannelSnafu)?,
+                )
+                .map_ok(drop)
+                .context(ForwardStdinSnafu)
+                .boxed(),
+            ])
+            .await
+            .0?;
+            Ok(String::new())
+        }
+        .instrument(span)
+        .await
+    }
+}
+
+fn generate_debug_container_name() -> String {
+    let mut rng = rand::thread_rng();
+    let mut name = "sble-debug-".to_string();
```

Comment:
suggestion: Let's use the full name: `stackable-debug-abcde`.

Reply:
1567a8da411f86178862147014e7e1e14ca068a1

Reply:
335e1ac0c4fa14b68c223c487704b37bbde1f171